### PR TITLE
update the helpers

### PR DESCRIPTION
**Pull request by kmuthukumar23**

### CODE REVIEW

File: cypress/integration/home_page_spec.js [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
     });
 
-    it('invoke function', function() {
+    it('invoke function', function () {
         curDropDown.invoke("showPopup");
     });
 
+    it('Event handling', function () {
+        curDropDown.eventHandler('change', function (e) {
+            debugger;
```
Comment (SridharSathaSivam) [inline cypress/integration/home_page_spec.js]: remove the debugger

File: package.json [hunk 2]
```diff
@@ -17,6 +17,8 @@
     },
     "homepage": "https://github.com/SyncfusionExamples/ej2-cypress-helperpoc#readme",
     "devDependencies": {
-        "cypress": "3.1.4"
+        "cypress": "3.1.4",
+        "@syncfusion/ej2-base": "*",
```
Comment (SridharSathaSivam) [inline package.json]: add the specific version

Comment (SridharSathaSivam) [reply package.json]: please resolve this

**Review by SridharSathaSivam — CHANGES_REQUESTED** [review]

File: cypress/integration/home_page_spec.js (hunk 1 above)
Comment (kmuthukumar23) [reply cypress/integration/home_page_spec.js]: Debugger Removed

File: package.json (hunk 2 above)
Comment (kmuthukumar23) [reply package.json]: Added specific version

**Pull request merged**